pikaraoke/app.py

```diff
@@ -707,6 +716,85 @@ def get_default_dl_dir(platform):
             return "~/pikaraoke-songs"
 
 
+@app.route("/audio_devices/")
+def get_audio_devices():
+    audio_devices = k.get_audio_devices()
+    bluetooth_devices = k.get_bluetooth_devices()
```

Comment:
Do these need to be wrapped behind the k object? Can you just call these funcitons directly from the audio_devices lib?